.gitignore

```diff
@@ -78,3 +78,6 @@ wp-tests-config.php
 
 # Files for local environment config
 /docker-compose.override.yml
+
+# Visual regression test diffs
+tests/visual-regression/specs/__image_snapshots__
```

Comment:
I'm not sure how, but I guess this also has to be added to the upstream SVN repo?

Reply:
That's a very good question 😅 hopefully someone knows the answer to.

Reply:
Yes, the folder will need to be excluded upstream when committing by updating `svn props`.

Reply:
Yes, and the changed svn props can also be committed afterwards, if easier. As far as I see `svn:ignore __image_snapshots__` will have to be added to the parent dir, `tests/visual-regression/specs`.